backend/src/routes/auth.js

```diff
@@ -85,53 +100,42 @@ track of this transaction
 to authenticate into other endpoints
 */
 async function handleLogin(req, res) {
```

Comment:
nit: If you're changing the route corresponding to this handler could you also update the handler name so that its more apparent that it matches to the route. Also why are we switching from POST to GET for this?

Reply:
Using GET because there's no request body--we pass in idToken via header instead

Also semantically, it's kinda like "retrieving" a session Id token

I guess none of these reasons are huge so if you want I can change back to POST.

Reply:
also handler name is totally my bad, not a nit at all xD